shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1085,25 +1091,15 @@ \subsection{Block Body Transition}
     {
       \var{txs} \leteq \bbody{block}
       &
-      \var{vk} \leteq \bissuer{bhb}
-      &
-      \var{hk} \leteq \hashKey{vk}
-      \\
-      \sigma \leteq \bsig{bhb}
+      \var{bhb} \leteq \bhbody\bheader{block}
       &
-      \var{bh} \leteq \bheader{block}
-      &
-      \var{bhb} \leteq \bhbody{bh}
-      \\
-      (\wcard,~\wcard,~\var{us})\leteq ls
-      &
-      \var{pp}\leteq\pps{us}
+      \var{hk} \leteq \hashKey\bissuer{bhb}
       \\~\\
-      \bBodySize{txs} < \fun{maxBBSize}~\var{pp}
+      \bBodySize{txs} = \bhbodysize{bhb}
```

Comment:
`bhbodysize` is the size of the block body as it is recorded in the block header, right?
Somehow this name is slightly confusing, it is also the accessor function to `bsize`, maybe it should be more similar to that name ?

Reply:
yea, I had a hard time coming up with a good name. it is indeed the recorded body size in the header. "block header body size". and yes, the accessor for `bsize`.

following the naming convention for the other variables in the header, it should be `bbsize`. but that looks like "block body size" :grimacing: 

Have any suggestions?

Reply:
yes, it's a bit inconvenient that we do have a block header body, too :-)
how about `hBbsize` ? for header block body size

Reply:
works for me!